### ISSUE TITLE
Fade function isn't working

### ISSUE THREAD

**Issue by michaeljtoth** [justinmc/jQuery-Open-Carousel#1]
Hi Justin

So far so good with your carousel! The default sliding animation works great. I'm trying to use the fade option, though, and when I set the transition to fade, my first image fades into white, and nothing comes up behind it. I've followed your set up exactly, except that I'm changing the default transition in the .js rather than passing the attribute in the css. You can see it live here: http://learn-more.sri.com/iom/

I hope you can help! Thanks!

Mike

**Comment by justinmc** [comment]
Hey Mike,

Thanks for pointing this out.  It looks like the problem is with the old version of jQuery you're using (1.4.2).  The easiest way to fix this is to update your jQuery to 1.4.3+ (now specified in the README), or try the fix I just committed (55cb63080ce65792b314cc73446b1a098b3e09ca) by updating your jquery.openCarousel.js file.

If you try my fix, be aware that it seems there are other reasons why this project can't fully support 1.4.2.  For the simple usage you linked above it should work though.

The fading problem occurred because 1.4.2 doesn't support callbacks on the fadeOut function, so the frames weren't moving and fading back in after fading out.  I just used a different syntax and it seems to work in my tests with 1.4.2. now for a simple carousel.

Let me know if this solved your problem and I'll close the issue.
